XCUITests/FxScreenGraph.swift

```diff
@@ -542,6 +546,18 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
         screenState.backAction = navigationControllerBackAction
     }
 
+    map.addScreenState(WebsiteDataSettings) { screenState in
+        screenState.gesture(forAction: Action.AcceptClearAllWebsiteData) { userState in
+            app.tables.cells["ClearAllWebsiteData"].tap()
+            app.alerts.buttons["OK"].tap()
+        }
+            screenState.gesture(forAction: Action.TapOnFilterWebsites) { userState in
+            app.swipeDown()
```

Comment:
Add here a comment that this is a workaround for the intermittent issue that the search field is not shown.